src/UI/Internal/Text.elm

```diff
@@ -57,6 +56,12 @@ type TextColor
     | ColorInherit
 
 
+type TextOverflow
+    = Ellipsize
+    | EllipsizeWithTooltip
+    | Wrap
```

Comment:
I see there isn't a `NoWrap` or `BreakWords`, but that's okay by now, when the need appears we add them